charts/stable/romm/questions.yaml

```diff
@@ -42,6 +42,33 @@ questions:
 # Include{containerBasic}
 # Include{containerAdvanced}
 
+  - variable: romm
+    group: App Configuration
+    label: Romm
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: auth
+          label: Auth Credentials
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: user
+                label: Auth User
+                schema:
+                  type: string
+                  required: true
+                  default: ""
+              - variable: pass
+                label: Auth Password
+                schema:
+                  type: string
+                  required: true
+                  private: true
+                  default: ""
+
```

Comment:
Hmm this might block upgrades, cause is  required + no default

Reply:
could i just set the values as admin + password?

Reply:
lets do that yes.